workers/unity/spatialos.unity.client.build.json

```diff
@@ -75,10 +75,11 @@
       "name": "Clean",
       "steps": [
         {
-          "name": "Dependencies",
+          "name": "Remove Codegen",
+          "command": "rm",
           "arguments": [
-            "worker_package",
-            "clean"
+            "-rf",
+            "./Assets/improbable/genereated/*"
```

Comment:
this path still has the typo in it ;)
change it to be the same as for the gamelogic build file